src/dynamodb/client.py

```diff
@@ -353,16 +250,17 @@ def bulk_insert_github_node_to_asana_id_mapping(
     )
 
 
-def DEPRECATED_bulk_insert_github_handle_to_asana_user_id_mapping(
-    gh_and_asana_ids: List[Tuple[str, str]]
-):
+def get_asana_domain_user_id_from_github_handle(github_handle: str) -> Optional[str]:
```

Comment:
will you be following up with a PR to migrate this out of the dynamodb client? given that we aren't interacting with dynamodb in this method?

Reply:
ooh great question! I was originally thinking of moving all of the top-level functions from `src/dynamodb/client.py` to something like `src/sgtm_state.py`. I figured the functions in `sgtm_state` could then call the instance/class methods on `src.dynamodb.client.DynamoDbClient` and/or on `src.s3.client.S3Client` (new file) -- what do you think?

My motivation was that it still feels useful to have all 'state' functions live in one file, and abstract the notion of SGTM state away from _where_ the state lives (dynamoDB vs. S3). Let me know what you think!

Reply:
It would be nice to abstract out the client-specific helpers to a generic helper so that the code doesn't need to know where its getting its data. We have a `src/github/helpers.py` file which just checks the Pull Request object to see if it has labels that could semantically fit but we can iterate on the location.

I did see https://github.com/Asana/SGTM/pull/166#issuecomment-2052750437 and I think the change is fine. Worth making a small announcement about in SGTM changelog/version bump/slack channels